src/main/java/hu/oe/nik/szfmv/automatedcar/Main.java

```diff
@@ -9,29 +10,23 @@
 
     private static final Logger LOGGER = LogManager.getLogger();
     private static final int CYCLE_PERIOD = 40;
-    // The window handle
+
     private Gui window;
-    private AutomatedCar car;
-    private World world;
+    private WorldManager worldManager;
 
     public static void main(String[] args) {
-
         new Main().run();
-
     }
 
     public void run() {
         init();
         loop();
-
     }
 
     private void init() {
-        // create the world
-        world = new World(5000, 3000);
-        // create an automated car
-        car = new AutomatedCar(20, 20, "car_2_white.png");
-        world.addObjectToWorld(car);
+        AutomatedCar car = new AutomatedCar(20, 20, "car_2_white.png");
```

Comment:
18:36:47.411 [main] ERROR hu.oe.nik.szfmv.automatedcar.model.WorldObject - Nem sikerült a képfájl betöltése! Fájl neve: car_2_white.png” 

De ha kiveszem a .png-et a string végéről, akkor csodával határos módon a hiba többé nem jelentkezik. (Az autó ugyan nem jelenik meg kirajzolva továbbra sem, ami enyhe visszalépés a skeleton óta. :yum: )
Felmerül a kérdés, hogy elindította-e valaki ezt a kódot mielőtt PR lett belőle.

Reply:
Szerintem kicsit szerencsétlenül oldottuk meg az egészet. Úgy kalkuláltunk, hogy mi implementáljuk a fájlok feldolgozását és majd a megjelenítés használja. Viszont elmaradt a test branches próba-demo, vele együtt az integrálás, így alakult ki ez a helyzet. Elég ciki, legközelebb jobban fogunk figyelni!

Igen, a hibát javítottam már (bár már úgy is mindegy :D), valóban az lehetett, hogy az utolsó PR előtt nem lett elindítva a branch.

@deseatnikov @kaptast 
Tanulság: mindig (!) indítsuk el az alkalmazást a fejlesztés végén, ne legyen elég az, hogy lefutottak a tesztek. Meg persze az se árt, ha be van configolva a log4j és figyeljük a logot. :D Illetve legközelebb 1-2 nappal demo előtt kell elkészülnünk mindennel, hogy legyen idő reviewzni.